### PR TITLE
Create a table in privileges for `Roles`

### PR DESCRIPTION
**Pull request by Andresbl123**
Create a new “roles” option, including a table. (This information is currently used in the AppMenu Default Story.

### CODE REVIEW

**Review by LinconOssa — APPROVED** [review]
Cumple con requerimientos

File: src/mocks/privileges/roles/rolesData.muck.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export const roles = [
```
Comment (carin91) [inline src/mocks/privileges/roles/rolesData.muck.ts:1]: Change the name of the file, rolesData.mock.ts

File: src/mocks/privileges/roles/rolesData.muck.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  {
+    id: "11",
+    code: 1121212,
+    nameRoles: "Incluir las opciones web 30,31 y 32",
+    aplicaicionRoles: "cooservunal",
```
Comment (carin91) [inline src/mocks/privileges/roles/rolesData.muck.ts:6]: Adjust the name for this key, aplicationRoles

File: src/mocks/privileges/roles/rolesData.muck.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
+export const roles = [
+  {
+    id: "11",
```
Comment (carin91) [inline src/mocks/privileges/roles/rolesData.muck.ts:3]: Change the example based on this examples:
https://senlinea.sharepoint.com/:x:/s/isetting221/EdnkwzE5dxpCjCDVZqnEPPgByLi_J40olmbVB05AoGn8sw?e=OfbsqY
image

File: src/mocks/privileges/roles/rolesData.muck.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    code: 1121212,
```
Comment (carin91) [inline src/mocks/privileges/roles/rolesData.muck.ts:4]: The backend service contains: (Código, Nombre, Descripción, Aplicación, Estado) and should be able to use "Activar", "Editar" y "Eliminar", Therefore, the id key chould be deleted, 
image

The keys for this entity are:
image

So the appropriate mock should be:
  {
    k_rol: 23123213,
    n_rol: "Analista de Crédito",
    n_uso: "Este es un analista de crédito"
    k_aplica : "Viabilizar crédito",
   i_activo: "0",
  },

**Review by carin91 — CHANGES_REQUESTED** [review]

**Review by carin91 — APPROVED** [review]

**Pull request merged**